src/lib/multiselect.directive.js

```diff
@@ -55,6 +55,7 @@
             self.groupOptions = {};
             self.optionsFiltered = {};
             self.filter = {};
+            self.limit = getSettingValue('limitTo', true) || 500;
```

Comment:
This default value probably belongs in the [global `amoMultiselectConfig` constant](https://github.com/namoscato/angular-multiselect/blob/332c4689c364544ca3860b7d5434c0e55071c0fe/src/lib/multiselect-config.constant.js).